Local build failing in config.core due to violation in schema\config-description-1.0.0.xsd

````
[INFO] openHAB Core ....................................... SUCCESS [01:10 min]
[INFO] openHAB Core :: BOM ................................ SUCCESS [  0.178 s]
[INFO] openHAB Core :: BOM :: Compile ..................... SUCCESS [  0.570 s]
[INFO] openHAB Core :: BOM :: Compile Model ............... SUCCESS [  1.217 s]
[INFO] openHAB Core :: BOM :: Runtime ..................... SUCCESS [  2.623 s]
[INFO] openHAB Core :: BOM :: Runtime Index ............... SUCCESS [  3.431 s]
[INFO] openHAB Core :: BOM :: Test ........................ SUCCESS [  0.200 s]
[INFO] openHAB Core :: BOM :: Test Index .................. SUCCESS [  0.495 s]
[INFO] openHAB Core :: Bundles ............................ SUCCESS [  8.437 s]
[INFO] openHAB Core :: Bundles :: Core .................... SUCCESS [ 25.001 s]
[INFO] openHAB Core :: Bundles :: Test .................... SUCCESS [  3.361 s]
[INFO] openHAB Core :: Bundles :: Configuration Core ...... FAILURE [  0.463 s]
...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:58 min
[INFO] Finished at: 2024-05-25T14:23:01+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.43.0:check (codestyle_check) on project org.openhab.core.config.core: The following files had format violations:
[ERROR]     schema\config-description-1.0.0.xsd
[ERROR]         @@ -1,165 +1,165 @@
[ERROR]         -<?xml version="1.0" encoding="UTF-8"?>\n
[ERROR]         -<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"\n
[ERROR]         -       xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"\n
[ERROR]         -       targetNamespace="https://openhab.org/schemas/config-description/v1.0.0">\n
[ERROR]         -\n
[ERROR]         -       <xs:element name="config-descriptions">\n
[ERROR]         -               <xs:complexType>\n
[ERROR]         -                       <xs:sequence>\n
[ERROR]         -                               <xs:element name="config-description" type="config-description:configDescription" minOccurs="1"\n
[ERROR]         -                                       maxOccurs="unbounded"/>\n
[ERROR]         -                       </xs:sequence>\n
[ERROR]         -               </xs:complexType>\n
[ERROR]         -       </xs:element>\n
[ERROR]         -\n
[ERROR]         -       <xs:element name="config-description" type="config-description:configDescription"/>\n
[ERROR]         -\n
[ERROR]         -       <xs:simpleType name="idRestrictionPattern">\n
[ERROR]         -               <xs:restriction base="xs:string">\n
[ERROR]         -                       <xs:pattern value="[A-Za-z0-9\-_]+"/>\n
[ERROR]         -               </xs:restriction>\n
[ERROR]         -       </xs:simpleType>\n
[ERROR]         -\n
[ERROR]         -       <xs:simpleType name="uriRestrictionPattern">\n
[ERROR]         -               <xs:restriction base="xs:string">\n
[ERROR]         -                       <xs:pattern value="[A-Za-z0-9\-_]+(:[A-Za-z0-9\-_]+){1,2}"/>\n
[ERROR]         -               </xs:restriction>\n
[ERROR]         -       </xs:simpleType>\n
[ERROR]         -\n
[ERROR]         -       <xs:complexType name="configDescription">\n
[ERROR]         -               <xs:sequence>\n
[ERROR]         -                       <xs:element name="parameter-group" type="config-description:parameterGroup" minOccurs="0"\n
[ERROR]         -                               maxOccurs="unbounded"/>\n
[ERROR]         -                       <xs:element name="parameter" type="config-description:parameter" minOccurs="0" maxOccurs="unbounded"/>\n
[ERROR]         -               </xs:sequence>\n
[ERROR]         -               <xs:attribute name="uri" type="config-description:uriRestrictionPattern" use="optional"/>\n
[ERROR]         -       </xs:complexType>\n
[ERROR]         -\n
[ERROR]         -       <xs:complexType name="parameter">\n
[ERROR]         -               <xs:all>\n
[ERROR]         -                       <xs:element name="context" type="xs:string" minOccurs="0"/>\n
[ERROR]         -                       <!-- deprecated element "required" -->\n
[ERROR]         -                       <xs:element name="required" type="xs:boolean" default="false" minOccurs="0"/>\n
[ERROR]         -                       <xs:element name="default" type="xs:string" minOccurs="0"/>\n
[ERROR]         -                       <xs:element name="label" type="xs:string" minOccurs="0"/>\n
[ERROR]         -                       <xs:element name="description" type="xs:string" minOccurs="0"/>\n
[ERROR]         -                       <xs:element name="options" type="config-description:optionsType" minOccurs="0"/>\n
[ERROR]         -                       <xs:element name="limitToOptions" type="xs:boolean" minOccurs="0"/>\n
[ERROR]         -                       <xs:element name="filter" type="config-description:filterType" minOccurs="0"/>\n
[ERROR]     ... (282 more lines that didn't fit)
[ERROR] Run 'mvn spotless:apply' to fix these violations.
```

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,3 +1,4 @@
 *.java text=auto
 *.xml text=auto
 *.xml_gen text=auto
+*.xsd text=auto

